guides/doc-Administering_Red_Hat_Satellite/topics/Renaming_a_Capsule_Server.adoc

```diff
@@ -44,7 +44,7 @@ Ensure that you enter the full path to the `.tar` file.
 +
 Ensure that you enter the full path to the `.tar` file.
 
-. Optional: If you have created a custom certificate for {SmartProxyServer}, on {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the `{certs-generate}` command returns. For more information, see {BaseURL}installing_capsule_server/installing-capsule-server#deploying-a-custom-ssl-certificate-to-capsule-server_capsule[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _Installing {SmartProxyServer}_.
+. Optional: If you have created a custom certificate for {SmartProxyServer}, on {SmartProxyServer}, to deploy the certificate, enter the `{foreman-installer}` command that the `{certs-generate}` command returns. For more information, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-capsule-server_capsule[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _Installing {SmartProxyServer}_.
```

Comment:
https://docs.theforeman.org/master/Installing_Proxy_on_Red_Hat/index-foreman.html#deploying-a-custom-ssl-certificate-to-capsule-server_capsule

Here too